edge-modules/SimulatedTemperatureSensor/docker/windows/arm32v7/base/Dockerfile

```diff
@@ -12,9 +12,8 @@ RUN curl -SL %DOTNET_DOWNLOAD_URL% --output dotnet.zip \
 USER ContainerAdministrator
 # In order to set system PATH, ContainerAdministrator must be used
 RUN setx /M PATH "%PATH%;C:\Program Files\dotnet"
-# Add an unprivileged user account for running the module
-RUN net user /add moduleuser
-USER moduleuser
+
+USER ContainerUser
```

Comment:
Interesting. Why do we need an explicit `USER` entry only for this image?

Reply:
Because we are switching for ContainerAdministrator for the setx command above. 
So now instead of moving to moduleuser (as it was), we move to ContainerUser.

Reply:
Quick clarification on the PR description--by default containers run as ContainerAdministrator, hence why you need an explicit `USER ContainerUser`